[CN-1245][CN-1219]Update restricted installation steps (#198)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/get-started.adoc

[source,shell,subs="attributes"]
----
helm install operator hazelcast/hazelcast-platform-operator --version={operator-chart-version} -n hz-system \
    --set=createClusterScopedResources=false \ <1>
    --set=webhook.enabled=false \ <2>
    --set=enableHazelcastNodeDiscovery=false \ <3>
    --set=installCRDs=false \
    --set=watchedNamespaces="{ns-1, ns-2}"
----
--
====

> <1> Disabling `createClusterScopedResources` confines the operator to manage resources only within the specified namespaces, enhancing security and compliance.

Reviewer: ```suggestion
<1> Disabling `createClusterScopedResources` means that the management of resources by Operator is constrained to specified namespaces. This enhances both security and compliance.
```

> <2> `webhook.enabled=false` is necessary because enabling webhooks requires cluster-wide permissions, which conflicts with our restrictions against cluster-scoped resource creation.

Reviewer: ```suggestion
<2> Disabling `webhook.enabled` means that webhooks cannot be used. This is needed as the cluster-wide permissions required for webhooks conflict with our restrictions on cluster-scoped resource creation.
```

> <3> By setting `enableHazelcastNodeDiscovery` to false, the operator will not automatically discover nodes across all namespaces. This limits the use of `NODE_AWARE` in `highAvailabilityMode` and `NodePort` in `discoveryServiceType`, which depend on broader node discovery.

Reviewer: ```suggestion
<3> Disabling `enableHazelcastNodeDiscovery` means that Operator does not automatically discover nodes across all namespaces. This limits the use of `NODE_AWARE` in `highAvailabilityMode` and of `NodePort` in `discoveryServiceType`, both of which depend on broader node discovery.
```
